### ISSUE TITLE
Top Bar Font Size

### ISSUE THREAD

**Issue by MONO77** [vinceliuice/Orchis-theme#157]
Hi, I am wondering if there is a "best-practice" way to change the font size in top bar. Normally, I should change `$THEME_ROOT/gnome-shell/gnome-shell.css`, but I think in this file, it is said that `/* This stylesheet is generated, DO NOT EDIT */`, so I am wondering where I should edit.

**Comment by johndzxu** [comment]
Yes, I'm in the same situation. Also, I would also want to know how to configure the icon size in the top panel.
